Add documentation for different logpipeline comparison (#661)
kyma-project/community: concepts/observability-configurable-logging/05-custom-fluentbit-operator.md

To configure Fluent Bit, the user must create a new CR regarding to the CRD of this operator. Then, this operator will notice the new or changed CR, and will create or update a Config Map for Fluent Bit. Before the Config Map is applied, the operator uses the `dry-run` feature of Fluent Bit to validate the new configuration. If the check was successful, the new Config Map is applied and Fluent Bit is restarted.

## Overview Log routing scenarios and input configuration

The possible log routing scenarios offer the following combinations with the Fluent Bit input configurations:

|                 | Scenario 1 | Scenario 2 | Scenario 3 |
|-----------------|------------|------------|------------|
| Setup 1         | x          | x          | x          |
| Setup 2         | x          | x          |            |
| Setup 3         | x          | x          |            |
| Setup 4         |            |            | x          |

The following sections describe the details of the log routing scenarios and input configurations.
For details, see the [comparison with different scenarios of log pipeline routing](./07-multiple-logpipeline-investigation.md).

> After trying out various possibilities of log pipeline, we decided to choose go with [Setup 3c ](./07-multiple-logpipeline-investigation.md/#setup-3c) for following reasons:

Author: fixed

> 1. If one output fails, Fluent Bit will still push logs to the other output.

Reviewer: If the order of steps doesn't matter, use an unordered list.

```suggestion
- If one output fails, Fluent Bit will still push logs to the other output.
```

> 2. The rewrite tag with filesystem buffer enables the logs to buffered as chunks in case of failure of one outputs.

Reviewer: ```suggestion
- The rewrite tag with filesystem buffer enables the logs to buffered as chunks if one output fails.
```

> however, in case of failure of output for prolonged period of time and the filesystem buffer being full, there is loss of logs seen.

Reviewer: ```suggestion
However, if an output fails for a prolonged period of time and the filesystem buffer is full, logs are lost.
```

> We still have to decide how the rewrite tag should be configured. There are following proposals:

Reviewer: ```suggestion
We still have to decide how the rewrite tag should be configured. There are the following proposals:
```
